ddtrace/contrib/celery/app.py

```diff
@@ -1,9 +1,18 @@
+import celery
 from celery import signals
 
 from ddtrace import Pin
 from ddtrace import config
 from ddtrace.pin import _DD_PIN_NAME
+from ddtrace.vendor import wrapt
```

Comment:
We can remove this import and instead use `trace_utils.wrap` just for consistency 😄